google-cloud-trace/lib/google/cloud/trace/faraday_middleware.rb

```diff
@@ -20,11 +20,36 @@ module Google
   module Cloud
     module Trace
       class FaradayMiddleware < Faraday::Middleware
+        ##
+        # # Trace FaradayMiddleware
+        #
+        # A faraday middleware that setup request/response labels for trace.
+        #
+        # ## Installing
+        #
+        # To use this middleware, simply install it in your middleware stack.
+        # Here is an example configuration enable the Trace middleware:
+        #
+        # ```ruby
+        # conn = Faraday.new(:url => 'http://sushi.com') do |faraday|
+        #   # enable cross project tracing with option to true
+        #   faraday.use Google::Cloud::Trace, enable_cross_project_tracing: true
+        #   faraday.request  :url_encoded             # form-encode POST params
+        #   faraday.response :logger                  # log requests to $stdout
+        #   faraday.adapter  Faraday.default_adapter  # make requests with Net::HTTP
+        # end
+        # ```
+
+        def initialize(app, options: {})
```

Comment:
Is this correct, or should it be
```ruby
def initialize app, options = {}
```
I'm not 100% sure about the Faraday middleware constructor interface, but if it's anything like Rack middleware, I'd expect it to be normal arguments rather than an explicit `options` keyword argument.

Reply:
Yeah, that was my fault, was changing the code from logger without really testing it 😂 
https://github.com/lostisland/faraday/blob/master/lib/faraday/response/logger.rb

Just updated it and ran the test in with my local project, should use following instead:
```
def initialize app, enable_cross_context: false
```